packages/wrangler/src/__tests__/type-generation.test.ts

```diff
@@ -112,6 +131,7 @@ const bindingsConfigMock: Omit<
 		bindings: [
 			{ name: "DURABLE_TEST1", class_name: "Durability1" },
 			{ name: "DURABLE_TEST2", class_name: "Durability2" },
+			{ name: "DURABLE_TEST3", class_name: "Durability3" },
```

Comment:
What about a Durable Object defined in another Worker?
Can you add a test to ensure that this works correctly?
Particularly in the case where there is a matching file in the local Worker which may be picked up incorrectly.

Reply:
Test added!